packages/cli/src/requests.ts

```diff
@@ -386,3 +386,19 @@ export type BinaryDataRequest = AuthenticatedRequest<
 		mimeType?: string;
 	}
 >;
+
+// ----------------------------------
+//              /sso
+// ----------------------------------
+
+export declare namespace SSO {
+	export type AcsRequest = AuthenticatedRequest<
```

Comment:
[from what I can tell](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/middlewares/auth.ts#L100), this isn't supposed to be an authenticated request.

@flipswitchingmonkey can you please confirm this?